src/librustc/dep_graph/query.rs

```diff
@@ -36,7 +36,7 @@ impl DepGraphQuery {
         }
 
         DepGraphQuery {
-            graph: graph,
+            graph,
             indices: indices
```

Comment:
hm, yes, regex should've probably been ` (\w+): \1,?\n` rather than ` (\w+): \1,`